README.md

```diff
@@ -1,2 +1,276 @@
-# example-application
-Example out-of-tree application that is also a module
+![Zephyr Project logo](https://github.com/zephyrproject-rtos/zephyr/blob/v1.14-branch/doc/images/Zephyr-Project.png)
+
+
+# Repository Overview
+
+The Zephyr Project is a scalable real-time operating system (RTOS) supporting
+multiple hardware architectures, optimized for resource constrained devices,
+and built with security in mind.
+
+The Zephyr build system supports integration of externally maintained
+projects called *modules*. This integration is configured through the
+settings in a *module.yml* file in a :file:`zephyr/` folder at the root
+of the repository.  See the [Zephyr Modules (External projects)][5]
+for a high-level overview of the build system module mechanism.
+
+
+## Scope
+
+The Zephyr eXample Application (ZXA) module is supplementary to the Zephyr
+repository.  This provides:
+
+1. A reference implementation of using the Zephyr module mechanism.
+
+1. A repository for holding example code and config files outside of the
+   Zephyr repository, including:
+
+   - `west` manifest (in the default location)
+   - projects/samples/tests
+   - applications/subsystems/drivers/interface declarations
+   - devicetree source files
+   - boards
+   - scripts
+
+1. A test case for CI to monitor Zephyr build system support for modules.
+
+
+   _WARNING:_ The Zephyr repository is intentionally
+   **independent of and unaware of this repository**.
+
+
+# Getting Started
+
+## Creating a New Module
```

Comment:
This repository serves both as an example of a manifest repository in star topology and for a Zephyr module. Please use repository instead of module.